src/tonuino.cpp

```diff
@@ -50,6 +50,8 @@ void Tonuino::setup() {
   }
 
   SM_tonuino::start();
+  digitalWrite(ampEnablePin, getLevel(ampEnablePinType, level::active));
```

Comment:
`ampEnablePinType` is not defined for the classic board, so you have to use

`#if defined ALLinONE || defined ALLinONE_Plus`

Reply:
Sure, you're right. That occurred to me right after I created the PR. I'll have another look later.